src/components/Elements/Basic.tsx

```diff
@@ -57,13 +57,15 @@ const Basic: FunctionComponent<Props> = (props) => {
     background: "#4c4c4c",
   };
 
+  const tooltipId = `rt-tooltip-${altId ?? Math.floor(Math.random())}`;
```

Comment:
`Math.floor(Math.random())` will always return 0 because `Math.random()` returns a value between 0 and 1. To get a unique number, you have to multiply by 100 or something. Alternatively, I think generating a random uuid via an npm package might be better. (i.e. https://www.npmjs.com/package/uuid).

Reply:
Good call, I'll go the uuid route. Wanted to avoid another dependency if I could, but that'll be a better solution that's worth it.